gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -1074,8 +1072,6 @@ public Builder megabytesCopiedPerChunk(Long megabytesCopiedPerChunk) {
        * Creates a {@code CopyRequest} object.
        */
       public CopyRequest build() {
-        checkNotNull(source);
```

Comment:
I think we could have left that here (though only checking source and targetId if you ignore my previous suggestion).